packages/component/src/Styles/StyleSet/ActivityButton.ts

```diff
@@ -0,0 +1,45 @@
+export default function createActivityButtonStyle() {
+  return {
+    '&.webchat__activity-button': {
+      alignItems: 'center',
+      appearance: 'none',
+      background: '#fff',
+      borderRadius: 4,
```

Comment:
```suggestion
      borderRadius: '4px',
```
To make units consistent in the file